test/unit/test_extension_wrapper_common.py

```diff
@@ -22,6 +25,24 @@ def filled_secrets(secrets) -> Secrets:
     return secrets
 
 
+@pytest.fixture
+def filled_saas_secrets(secrets) -> Secrets:
+    secrets.save(CKey.saas_url, "https://mock_saas.exasol.com")
+    secrets.save(CKey.saas_account_id, "w53lhsoifid794ms")
+    secrets.save(CKey.saas_database_name, "my_database")
+    secrets.save(CKey.saas_token, "xmfi58302lfj0ojf64ndk3ls")
```

Comment:
Should this secret be committed to the git repo?

Reply:
Or if these are mock secrets, maybe use a more obvious fake, like `token_123` and `account_123`.

Reply:
OK, will do.